tests/integration/test_charm.py

```diff
@@ -187,3 +188,45 @@ async def test_with_ingress(
     )
     assert response.status_code == 200
     assert "Hello, World!" in response.text
+
+
+@pytest.mark.parametrize(
+    "endpoint,db_name, db_channel",
+    [
+        (
+            "mysql/status",
+            "mysql-k8s",
+            "8.0/stable",
+        ),
+        (
+            "postgresql/status",
+            "postgresql-k8s",
+            "14/stable",
+        ),
+    ],
+)
+async def test_with_database(
+    flask_app: Application,
+    model: juju.model.Model,
+    get_unit_ips,
+    endpoint: str,
+    db_name: str,
+    db_channel: str,
+):
+    """
+    arrange: build and deploy the flask charm.
+    act: deploy the ingress, configure it and relate it to the charm.
```

Comment:
Is this accurate? Is this test for the database or ingress?

Reply:
Fixed